### PR TITLE
EASY-2450: custom license in header bij file download vanuit easy-download

### PR DESCRIPTION
**Pull request by vesaakerman**
Fixes EASY-2450

#### When applied it will
* add `license key` and `license title` into `AuthorisationItem`

#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-auth-info                      | [PR#33](https://github.com/DANS-KNAW/easy-auth-info/pull/33)     | 
easy-download                      | [PR#42](https://github.com/DANS-KNAW/easy-download/pull/42)     | ..

### CODE REVIEW

File: src/main/scala/nl.knaw.dans.easy.solr4files/components/AuthorisationItem.scala [hunk 1]
```diff
@@ -32,7 +32,9 @@ case class AuthorisationItem(itemId: String,
                              owner: String,
                              dateAvailable: DateTime,
                              accessibleTo: RightsFor.Value,
-                             visibleTo: RightsFor.Value
+                             visibleTo: RightsFor.Value,
+                             licenseKey: String,
+                             licenseTitle: String
```
Comment (rvanheest) [inline src/main/scala/nl.knaw.dans.easy.solr4files/components/AuthorisationItem.scala]: one comma makes the next diff look much more beautiful 😜 

```suggestion
                             licenseTitle: String,
```

File: src/test/scala/nl.knaw.dans.easy.solr4files/components/FileItemSpec.scala [hunk 2]
```diff
@@ -50,7 +50,9 @@ class FileItemSpec extends TestSupportFixture {
       owner = "someone",
       dateAvailable = DateTime.now,
       accessibleTo = RESTRICTED_GROUP,
-      visibleTo = RESTRICTED_GROUP
+      visibleTo = RESTRICTED_GROUP,
+      licenseKey = "http://dans.knaw.nl/en/about/organisation-and-policy/legal-information/DANSGeneralconditionsofuseUKDEF.pdf",
```
Comment (rvanheest) [inline src/test/scala/nl.knaw.dans.easy.solr4files/components/FileItemSpec.scala]: please don't use the old DANS_LICENSE url

Comment (rvanheest) [inline src/test/scala/nl.knaw.dans.easy.solr4files/components/FileItemSpec.scala]: please don't use the old DANS_LICENSE url

**Review by rvanheest — COMMENTED** [review]
I was a little confused at first, but then I realized this service receives the extra fields, but then doesn't use them... It's just that you're over-fetching from `easy-auth-info` 😉

**Review by jo-pol — COMMENTED** [review]
no additional opinion

**Pull request merged**